flint.ui/src/views/flint/ConfigurationsPoint.vue

```diff
@@ -112,110 +115,128 @@ export default {
+        content: 'Use the slider to set the Pool 2 value.',
+        params: {
+          placement: 'top'
+        }
       },
-      pool3: {
-        min: 0.0,
-        max: 400.0,
-        interval: 5.0,
-        value: 100.0
+      {
+        target: '[data-v-step="4"]',
+        content: 'Use the slider to set the Pool 3 value.',
+        params: {
+          placement: 'top'
+        }
       },
-      myOptions: {
-        useKeyboardNavigation: true,
-        labels: {
-          buttonSkip: 'Skip tour',
-          buttonPrevious: 'Previous',
-          buttonNext: 'Next',
-          buttonStop: 'Finish'
+      {
+        target: '[data-v-step="5"]',
+        content: 'Click on the RUN button to start the simulation.',
+        params: {
+          placement: 'top'
         }
       },
-      steps: [
-        {
-          target: '[data-v-step="1"]',
-          content: 'Set the Start and End date of simulation from the date picker.',
-          params: {
-            placement: 'bottom'
-          }
-        },
-        {
-          target: '[data-v-step="2"]',
-          content: 'Use the slider to set the Pool 1 value.',
-          params: {
-            placement: 'top'
-          }
-        },
-        {
-          target: '[data-v-step="3"]',
-          content: 'Use the slider to set the Pool 2 value.',
-          params: {
-            placement: 'top'
-          }
-        },
-        {
-          target: '[data-v-step="4"]',
-          content: 'Use the slider to set the Pool 3 value.',
-          params: {
-            placement: 'top'
-          }
-        },
-        {
-          target: '[data-v-step="5"]',
-          content: 'Click on the RUN button to start the simulation.',
-          params: {
-            placement: 'top'
-          }
-        },
-        {
-          target: '[data-v-step="6"]',
-          content: 'Click on this button to see the Point output.',
-          params: {
-            placement: 'top'
-          }
+      {
+        target: '[data-v-step="6"]',
+        content: 'Click on this button to see the Point output.',
+        params: {
+          placement: 'top'
         }
-      ]
-    }
-  },
-  mounted: function () {
-    this.$tours['MyTour'].start()
-  },
-  methods: {
-    finalPoolValues() {
-      if (
-        this.pool1.value.toFixed(1) === this.pool2.value.toFixed(1) &&
-        this.pool2.value.toFixed(1) === this.pool3.value.toFixed(1)
-      ) {
-        console.log('Config values not changed!!')
       }
+    ]
 
-      this.$store.commit('set_pointConfig_Pool_1', this.pool1.value.toFixed(1))
-      this.$store.commit('set_pointConfig_Pool_2', this.pool2.value.toFixed(1))
-      this.$store.commit('set_pointConfig_Pool_3', this.pool3.value.toFixed(1))
+    function finalPoolValues() {
+      if (pool1.value.pool_value === 100 && pool2.value.pool_value === 100 && pool3.value.pool_value === 100) {
+        console.log('Pool values are probably unchanged!')
```

Comment:
Currently, the pools go back to 100 when we refresh the page. But in feat/gcbm-simulation-editor branch, I have added vue-persist. So, the initial values will be the same as last time (if changed).

Finding if the pool values are changed or not, in that case, won't work with just a condition. We'd have to keep a copy of the initial values in a var and compare the current values (pool1.value.pool_value, etc. with that object).

Do we need this? This stops updating the store unnecessarily, but the request to the server is made every time anyway, and updating the store isn't that expensive.

We can keep this for now, but what do you think about implementing a function to check if the store state has changed when the run button is clicked?
We'd need to keep a copy of the store when the run is clicked for the first time, and if a copy exists, it'll be compared with the current state, and the run function will only continue if there's a change. This will be needed in the GCBM editor too since the GCBM simulation is costly (takes about 15-20 mins).

Right now, we can change this statement to something more definite, and remove the `probably` :)

If you want you can work on creating a general function for checking if the state is changed or create a new issue for it :)

Reply:
Good points. I had thought about these and was waiting for your comments. 

The addition of `vuex-persist` changes things a bit. I gave a bit of thought to the solution. Here's what I think:

> Do we need this?

I think it'd be a good design choice. So yes. Especially since you mentioned that the simulations take up a non-trivial amount of time. And we can't realistically have a performance bottleneck at the store operations anyway.

>We'd need to keep a copy of the store when the run is clicked for the first time, and if a copy exists, it'll be compared with the current state, and the run function will only continue if there's a change.

This is good - but it also means that the user can only use *Run* once with a specific set of pool values and they get locked in. e.g if they intentionally want to run the Pool simulation with the exact values. To fight this problem, an average user will probably move the slider, re-adjust it back to the original value so they can continue running the simulation. That sounds sub optimal.

Keeping these in mind, here's my solution:
## Solution
1. Compare the current values of the pool sliders with the previous values in store.
2. Display a modal which gives a warning if the pool values are unchanged.

This modal can have an optional checkbox which says something like "Do not ask again" in case if the user does not want to get bothered with the warnings.

I will create a new issue for this, and update it with new information in case if we happen to have a conversation over Slack (And also because it's out of the scope of this PR). I think we can keep the check I've implemented for now. 

Let me know what you think!

Reply:
Hi, sorry for the late reply on this.
> if they intentionally want to run the Point simulation with the exact values
I don't see the point of this. If the simulation output is going to be the exact same if the values are not changed, then what would be the use of allowing this action? It's not much in the Point and RothC runs because they're not as computationally expensive as the GCBM run. 

Is it really needed?

Reply:
I gave it a bit of a thought.

You're right. I can implement your solution in another PR.

Reply:
@Crystalsage Sure!